.github/workflows/go.yml

```diff
@@ -0,0 +1,24 @@
+name: Go unit tests
+
+on: 
+  pull_request:
+    paths:
+      - '**.go'
+      - go.mod
+      - go.sum
+
+jobs:
+  gotest:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Go
+      uses: actions/setup-go@v5
+      with:
+        go-version-file: go.mod
+        check-latest: true
+
+    - name: Test
+      run: go test -v ./...
```

Comment:
We should use `make ci` here, because it also computes coverage. But I'm not sure the system we had in place still works (I bet it doesn't).

Reply:
There's probably some github action for that.